.github/metadata.py

```diff
@@ -85,6 +85,8 @@
 - `mypy` [rejected] at https://github.com/python/mypy/issues/13294
 """
 
+# pylint: disable=fixme,no-name-in-module,too-many-public-methods
```

Comment:
I am not fond of these special `# pylint` instructions everywhere in the code because I rely exclusively on `ruff`. But the way `ruff` is fed its config is not solved yet: https://github.com/kdeldycke/workflows/issues/659

So yeah, why not. These are inoffensive and I am ok to have them around if it keep you happy to send PRs! 😊

Reply:
And I am ok not including them in the code, but whiny logs are ignored logs. I prefer to quiet specific issues on a case/file basis and not globally.  Turning off issues globally seems overkill, as it may hide valid issues that should be looked into/addressed.

But perhaps the point of the `fixme` issue is to **not** ignore it, but to fail the lint run?